napari/_app_model/_app.py

```diff
@@ -32,7 +30,7 @@ def __init__(self, app_name=APP_NAME) -> None:
 
     @classmethod
     def get_app_model(cls, app_name: str = APP_NAME) -> NapariApplication:
-        return Application.get_app(app_name) or cls()  # type: ignore[return-value]
+        return cls()  # type: ignore[return-value]
```

Comment:
I think this needs to be renamed not removed here:
```suggestion
        return return Application.get_app_model(app_name) or cls()  # type: ignore[return-value]
```

Reply:
I debated that. It seemed a little odd to me to have Application.get_app_model in the get_app_model for a class derived from Application. Let's give your idea a shot since just `cls()`didn't feel right to me either.

Reply:
@willingc Keep looks like I led you astray. I'm sorry!